api/src/pcapi/routes/backoffice/users/forms.py

```diff
@@ -46,17 +46,21 @@ class UnsuspendUserForm(FlaskForm):
     comment = fields.PCOptCommentField("Commentaire interne optionnel")
 
 
-def get_toggle_suspension_args(user: users_models.User, suspension_type: SuspensionUserType | None = None) -> dict:
+def get_toggle_suspension_args(
+    user: users_models.User,
+    suspension_type: SuspensionUserType | None = None,
+    required_permission: perm_models.Permissions | None = None,
+) -> dict:
```

Comment:
suggestion: rendre obligatoire de passer suspension_type en argument nomé
```suggestion
def get_toggle_suspension_args(
    user: users_models.User,
    *,
    suspension_type: SuspensionUserType | None = None,
    required_permission: perm_models.Permissions | None = None,
) -> dict:
```

Reply:
Cool j'ai appris un truc en plus :-)

Reply:
tu peux aussi forcer que certains arguments soient passer en `*args` avec `/` il me semble (il y a une tres bonne explication dans la doc python mais je ne la retrouve plus)

Reply:
trouvée ! https://docs.python.org/3/tutorial/controlflow.html#special-parameters